presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,436 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+   and the maximum running queries.
+
+4. Specify the file in the Presto configuration: In the ``etc`` directory of your Presto installation,
+   create a file named ``resource-groups.properties``. In this file, add a line that specifies the location of
+   your JSON file: ``resource-groups.config-file = <file_path>``, where ``<file_path>`` is the path to your JSON file.
+
+5. Restart the Presto server: After creating the JSON file and pointing to it in the Presto configuration,
+   restart your Presto server. The new resource groups will take effect after the restart.
+
+.. code-block:: text
 
     resource-groups.configuration-manager=file
-    resource-groups.config-file=etc/resource_groups.json
+    resource-groups.config-file=etc/resource-groups.json
 
-Change the value of ``resource-groups.config-file`` to point to a JSON config file,
-which can be an absolute path, or a path relative to the Presto data directory.
+Database Resource Group Manager
+-------------------------------
+The Database Resource Group Manager in PrestoDB is a way to manage resources using a relational database.
 
-Resource Group Properties
--------------------------
+To set up a Database-based Resource Group Manager:
 
-* ``name`` (required): name of the group. May be a template (see below).
+1. Create the file ``etc/resource-groups.properties``.
 
-* ``maxQueued`` (required): maximum number of queued queries. Once this limit is reached
-  new queries will be rejected.
+2. Set the ``resource-groups.configuration-manager`` property to ``db``.
 
-* ``hardConcurrencyLimit`` (required): maximum number of running queries.
+3. Set up a relational database: The database should be accessible by Presto.
+   It will be used to store the configurations of your resource groups.
 
-* ``softMemoryLimit`` (required): maximum amount of distributed memory this
-  group may use before new queries become queued. May be specified as
-  an absolute value (i.e. ``1GB``) or as a percentage (i.e. ``10%``) of the cluster's memory.
+4. Create tables for resource groups and selection rules: You need to create tables in the
+   database that will store the definitions of your resource groups and the rules for selecting
+   the appropriate resource group for a given query.
 
-* ``softCpuLimit`` (optional): maximum amount of CPU time this
-  group may use in a period (see ``cpuQuotaPeriod``) before a penalty will be applied to
-  the maximum number of running queries. ``hardCpuLimit`` must also be specified.
+5. Specify the database in the Presto configuration: In the ``etc`` directory of your Presto installation,
+   create a file named ``resource-groups.properties``. In this file, add a line that specifies
+   the JDBC URL of your database: ``resource-groups.config-db-url = <jdbc_url>``, where ``<jdbc_url>`` is the
+   JDBC URL of your database.
+
+The database resource group manager loads the configuration from a relational database. With the
```

Comment:
This paragraph is good! Suggest deleting the first sentence "The database resource group manager loads the configuration from a relational database. "

because the second sentence "With the Database Resource Group Manager, changes to the configuration in the database take effect immediately and do not require a restart of the Presto server." 

explains to the reader why there is not "Restart the server" step here - they don't need to restart.